incrementals-publisher/lib/github.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * This module provides some helpers for working with GitHub for the
+ * incrementals publishing
+ */
+
+const GitHubApi = require('@octokit/rest');
+
+const GITHUB_TOKEN = process.env.GITHUB_TOKEN || 'invalid-dummy-secret';
+
+module.exports = {
+  commitExists: async (owner, repo, sha) => {
+    let github = new GitHubApi();
+    /* The function must have this Application Setting already created */
```

Comment:
What is an “Application Setting”?

You can authenticate to GitHub using a personal access token and `BASIC` auth. Does this code block that mode?